shard_to_single_vcf.py

```diff
@@ -0,0 +1,65 @@
+## THIS SCRIPT NEEDS SSDs!!!!
+import argparse
+
```

Comment:
I can't actually tell based on the blank lines guidance from pep8, but not sure you need a line here

Reply:
So this is a pep8 recommendation,  
```Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

Reply:
oh nice, thank you!